src/common/models/Projection.ts

```diff
@@ -288,14 +294,17 @@ export class Projection {
       metricsTimeseries,
     );
     this.vaccinations =
-      this.vaccinationsInfo?.ratioInitiatedSeries ||
+      this.vaccinationsInfo?.ratioFall2022BivalentBoosterSeries ||
```

Comment:
Hrm.  This seems confusing to me.  If you're going to rework this I think it would make more sense to get rid of `vaccinations` (in favor of `vaccinationsInitiated`) and create `vaccinationsFall2022BivalentBoosterDose` or similar.  I don't think putting the bivalent dose into "vaccinations" makes sense.

I'd also be fine with minimizing churn and leaving "vaccinations" as vaccinations initiated indefinitely.

Reply:
Yeah, that's a good point. 

My initial hope was that since `vaccinations` is the "default" vaccine metric, putting the bivalent data in that variable would propagate it through the site and minimize the number of updates needed. Unfortunately, a lot of changes were still necessary -- I'll look to rework this as you've outlined